third_party/ssc/ssc/cmod_pvwattsv5.cpp

```diff
@@ -644,18 +644,24 @@ static var_info _cm_vtab_pvwattsv5_1ts_outputs[] = {
 
 class cm_pvwattsv5_1ts : public cm_pvwattsv5_base
 {
+private:
+    bool system_inputs_are_setup;
 public:
 
     cm_pvwattsv5_1ts()
     {
         add_var_info(_cm_vtab_pvwattsv5_1ts_weather);
         add_var_info(_cm_vtab_pvwattsv5_common);
         add_var_info(_cm_vtab_pvwattsv5_1ts_outputs);
+        system_inputs_are_setup = false;
     }
 
     void exec()
     {
-        setup_system_inputs();
+        if (!system_inputs_are_setup) {
+            setup_system_inputs();
+            system_inputs_are_setup = true;
+        }
         double ts = as_number("time_step");
```

Comment:
This is probably the biggest win. Using the 1-timestep model from SSC caused the panels to be initialized every timestep. This isn't a design flaw on their end because they take the unchanging properties of the PV system as inputs each timestep, so it _could_ need to be reinitialized, but we know we're not changing those, so I hacked it to only initialize the first time.